test/knex.test.js

```diff
@@ -259,6 +260,77 @@ describe('transaction', function () {
 
     throw new Error('expected the call to throw')
   })
+
+
+  it('adopt-commit', async () => {
+
+    let foo1_id
+
+    const trx = await knex(DbConfigPG).transaction()
+    const txseneca  = await si.entity.adopt({handle:trx})
+
+    //I'm using the same Knex instance....
+    const foosSave = await txseneca.entity('foo').data$({p1:'t1'}).save$()
+    foo1_id = foosSave.id
+    await txseneca.entity('foo').data$({p1:'t2'}).save$()
+    
+    //I'm checking the state of the transaction, I have not committed yet
+    //so isCompleted should be false and the list should be empty
+    const isCompleted1 = txseneca.client.isCompleted()
+    expect(isCompleted1).equal(false)
+    let foosList = await si.entity('foo').list$()
+    expect(foosList.length).equal(0)
+    
+    //I'm updating the second element, using the same Knex instance
+    await txseneca.entity('foo').
+    data$({ p1: 't4', id: foo1_id })
+    .save$()
+
+    //I have not committed yet, so the list should be empty
+    let foosUpdate = await si.entity('foo').list$()
+    expect(foosUpdate.length).equal(0)
+
+    //I'm removing the first element, using the same Knex instance
+    await txseneca.entity('foo')
+    .data$({ id: foo1_id })
+    .remove$()
+
+    //I'm committing the transaction, using the same Knex instance
+    //So I'm expecting the first element to be removed and the second
+    //only element available to be t2
+    const tx1 = await txseneca.entity.end()
+    let foosCommit = await si.entity('foo').list$()
+    expect(foosCommit.length).equal(1)
+    expect(foosCommit[1].p1).equal('t2')
+
+    //I'm checking the state of the transaction, I have committed
+    //so isCompleted should be true
+    const isCompleted2 = tx1.client.isCompleted()
+    expect(isCompleted2).equal(true)
+
+
```

Comment:
add tests to verify data updated only with knex

Reply:
I did this at the beginning of the tests and created a separate test just for that. Check if everything is ok.